web/carousel/src/components/PasswordForm/PasswordForm.module.css

```diff
@@ -1,5 +1,15 @@
-.ErrorMessage {
-    color: gray;
-    font-size: small;
-    width: 500px;
-}
\ No newline at end of file
+.Span {
+    display: flex;
+    height: 13px;
+    margin-left: 10px;
+}
+
+.PasswordCheck {
+    height: 8px;
+    font-size: 6px;
```

Comment:
We can make font-size 9px, that would be readable enough i guess

Reply:
Sure!